cli/src/cli.rs

```diff
@@ -278,7 +278,16 @@ impl Command {
                     recovery_id,
                 )?;
 
-                let signer_public_key = signature.recover(&message)?;
+                let payload = [
+                    b"\x19Ethereum Signed Message:\n",
+                    message.len().to_string().as_bytes(),
+                    message.as_ref(),
+                ]
+                .concat();
```

Comment:
I think we don't always want to check an Ethereum typed signed message. Would be nice to have something like a `--raw` flag to check the signature of the message received without the prefix. Not sure if this should hash it or not.